app/views/errors/not_found.html.slim

```diff
@@ -1,12 +1,7 @@
 .govuk-grid-row
   .govuk-grid-column-two-thirds
-    h1.govuk-heading-xl Page not found
+    h1.govuk-heading-xl
+      = not_found.title
 
     p.govuk-body
```

Comment:
The multiple paragraphs in `body` are all going to be given `p.govuk-body` so there is no need to nest.